src/lib/Components/StickyTabPage/StickyTabPageFlatList.tsx

```diff
@@ -39,7 +43,9 @@ export const StickyTabPageFlatList: React.FC<StickyTabFlatListProps> = (props) =
   if (!stickyHeaderHeight) {
     throw new Error("invalid state, mounted flat list before stickyHeaderHeight was determined")
   }
-  const { tabIsActive } = useContext(StickyTabPageFlatListContext)
+  const { tabIsActive, tabSpecificContentHeight } = useContext(StickyTabPageFlatListContext)
+
+  const totalStickyHeaderHeight = Animated.add(stickyHeaderHeight, tabSpecificContentHeight ?? 0)
```

Comment:
I think this should better be
```suggestion
  const totalStickyHeaderHeight = Animated.add(stickyHeaderHeight, Animated.cond(Animated.defined(tabSpecificContentHeight),tabSpecificContentHeight,0) )
```

Reply:
is this `Animated.cond` and friends coming from RN or reanimated?

Reply:
@pvinis those are from reanimated! and thanks @MounirDhahri, let's go with that